include/dlaf/blas/tile_extensions.h

```diff
@@ -0,0 +1,52 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2021, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <blas.hh>
+
+#include "dlaf/common/callable_object.h"
+#include "dlaf/matrix/tile.h"
+
+#ifdef DLAF_WITH_CUDA
+#include <cublas_v2.h>
+
+#include "dlaf/cublas/error.h"
+#include "dlaf/util_cublas.h"
+#endif
+
+namespace dlaf {
+namespace tile {
+using matrix::Tile;
+
+template <class T>
+void subtract(const matrix::Tile<T, Device::CPU>& tile_a,
+              const matrix::Tile<const T, Device::CPU>& tile_b) {
+  DLAF_ASSERT(equal_size(tile_a, tile_b), tile_a, tile_b);
+  for (auto j = 0; j < tile_a.size().cols(); ++j)
+    blas::axpy(tile_a.size().rows(), T(-1), tile_b.ptr({0, j}), 1, tile_a.ptr({0, j}), 1);
+}
```

Comment:
What about a more generic "axpy like" implementation for tiles?
Note: I would invert the order of the arguments to use the same convention as in BLAS (the modified matrix is the last matrix argument).